src/tsickle.ts

```diff
@@ -16,6 +16,8 @@ import {toArray} from './util';
 
 export {convertDecorators} from './decorator-annotator';
 export {processES5} from './es5processor';
+export {FileMap, ModulesManifest} from './modules_manifest';
```

Comment:
I think there's also a ts.FileMap.  I think since anyone that uses tsickle also must use ts, it might be worth just using their type.  Though maybe the ES6 map thing I just mentioned means we don't want that...